.github/workflows/ci.yml

```diff
@@ -301,16 +335,22 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Install Rust ${{ env.rust_stable }}
-        uses: dtolnay/rust-toolchain@master
+        uses: dtolnay/rust-toolchain@stable
         with:
           toolchain: ${{ env.rust_stable }}
           target: ${{ matrix.target }}
-      - name: Install cross
-        uses: taiki-e/install-action@cross
-      - run: cross check --workspace --all-features --target ${{ matrix.target }}
+
+      - uses: taiki-e/setup-cross-toolchain-action@v1
+        with:
+          target: ${{ matrix.target }}
```

Comment:
Since tokio and tokio-* are libraries, cross-check probably does not require the installation of the toolchain.

Reply:
I reversed the change because `setup-cross-toolchain-action` does not support target `arm-linux-androideabi`.

Reply:
> Since tokio and tokio-* are libraries, cross-check probably does not require the installation of the toolchain.

Hmmm, that will work only if tokio does not depend on any C/C++ libraries.

Reply:
Let me have a try.

Reply:
Thanks, it worked.